.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-    - "0.10"
-    - "0.12"
     - "4"
```

Comment:
node.js 4.x is EOL so I think we can drop it if we want

Reply:
I've managed to leave it in there, but I disable outgoing request timeouts for keepalive on Node v4.x and earlier.